sigar_interface_test.go

```diff
@@ -256,6 +257,31 @@ var _ = Describe("Sigar", func() {
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	It("NetConn", func() {
```

Comment:
That's the convention used in the rest of the tests - they use `It("struct name")`. I don't like it either, but its just easier to make it consistent rather than fix it everywhere.